dist/templates/ovn-setup.yaml.j2

```diff
@@ -56,6 +56,13 @@ rules:
   - networkpolicies
   - statefulsets
   verbs: ["get", "list", "watch"]
+- apiGroups:
```

Comment:
this will need a change in CNO I believe. That will need to go in before this can go in or else we will break downstream merge. You may have already done that, and I just didn't see the PR?

Reply:
already merged https://github.com/openshift/cluster-network-operator/pull/891